design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -0,0 +1,201 @@
+type Query{
+    # See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/framework/attributes-metadata.md
+    pageSpecificCustomAttributes(
+        page_type: CustomAttributesPageEnum!
+    ): CustomAttributeMetadata
+}
+
+type Mutation {
+    createReturn(input: CreateReturnInput!): CreateReturnOutput @doc(description: "Create a new return.")
+    addReturnComment(input: AddReturnCommentInput!): AddReturnCommentOutput @doc(description: "Add a comment to an existing return.")
+    addReturnTracking(input: AddReturnTrackingInput!): AddReturnTrackingOutput @doc(description: "Add tracking information to the return.")
+    removeReturnTracking(input: RemoveReturnTrackingInput!): RemoveReturnTrackingOutput @doc(description: "Remove tracking information from the return.")
```

Comment:
> Remove tracking information from the return.

Since it's 1 item being removed from a collection, we could probably be more explicit in the description.

**Ideas**:
```suggestion
    removeReturnTracking(input: RemoveReturnTrackingInput!): RemoveReturnTrackingOutput @doc(description: "Remove a single tracked shipment from the return.")

```

```suggestion
    removeReturnTracking(input: RemoveReturnTrackingInput!): RemoveReturnTrackingOutput @doc(description: "Remove a single tracking number from the return.")

```